android/quest/src/main/java/org/smartregister/fhircore/quest/ui/questionnaire/QuestionnaireActivity.kt

```diff
@@ -247,7 +250,28 @@ open class QuestionnaireActivity : BaseMultiLanguageActivity(), View.OnClickList
   fun getQuestionnaireResponse(): QuestionnaireResponse {
     val questionnaireFragment =
       supportFragmentManager.findFragmentByTag(QUESTIONNAIRE_FRAGMENT_TAG) as QuestionnaireFragment
-    return questionnaireFragment.getQuestionnaireResponse()
+    return questionnaireFragment.getQuestionnaireResponse().apply {
+      // TODO this is required until require condition in [QuestionnaireResponseValidator] is fixed
```

Comment:
@maimoonak for the todo here. Do you have an idea of what needs to be done?  If do we have an issue for it? 

@f-odhiambo Can we add this to the SDK sprint?

Reply:
@dubdabasoduba @f-odhiambo https://github.com/google/android-fhir/issues/1713